PESDK Instagram Case Study Android/app/src/main/java/ly/img/pesdk_instagram_ui/app/instagram_ui/panel/TextPanel.java

```diff
@@ -304,7 +305,7 @@ public void unregisterLayoutHeightListener() {
     @Override
     public void onGlobalLayout() {
         if (this.getRootView() != null) {
-            Rect visibleDisplayFrame = RectRecycler.obtain();
+            Rect visibleDisplayFrame = ViewUtils.obtainScreenVisibleDisplayFrame(this.getRootView());
             this.getRootView().getWindowVisibleDisplayFrame(visibleDisplayFrame);
```

Comment:
This line overrides the calculations of ViewUtils.obtainScreenVisibleDisplayFrame